### DISCUSSION TITLE
Author/Article review

### DISCUSSION THREAD

**Discussion by iaiauto** [dailydotdev/daily#920] [Feature Requests]
Hello,

It is noticeable that some articles have rich content and are very informative, while others contain contents that are not relevant to the title, some seem to be AI-generated, some are even misleading, and some contain just some basic titles about the article main title with no deep info and are not informative at all.

I suggest that there would be a kind of review/scoring of the articles or even the source of the article/ original publishing site, using a simple 1-5 stars review method for example, as I think this will make daily.dev users to be able to filter quality content from useless ones, which will lead to a better time investment in daily.dev, instead of wasting some of the daily.dev users' time in low-quality content, and also will help the daily.dev management team to choose better sources of articles and remove the low-quality ones.

**Comment by idoshamun** [comment]
Hi @iaiauto,

Thanks for the fantastic suggestion. We will introduce downvote soon, allowing users to show their dissatisfaction with the content and allowing us to proceed accordingly. I think it should accommodate your needs, right?

**Reply by iaiauto** [reply]
Hello @idoshamun ,

Thanks for your kind reply, yes downvoting low-quality articles will do the job, hope you will be able too to identify low-quality sources/authors, waiting for this feature.

**Reply by nimrodkra** [reply]
@iaiauto downvoting is not live 🚀 [Check it out](https://app.daily.dev/posts/QK301jNUY)
